Terminal.Gui/Views/TableView/TableStyle.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+
+namespace Terminal.Gui {
+
+	public partial class TableView {
```

Comment:
Looks like `TableStyle` is still nested in `TableView`.

Reply:
Yup I have set `<WarningsAsErrors>CA1034</WarningsAsErrors>` and found a bunch more which I've changed to full classes in [915ec1b](https://github.com/gui-cs/Terminal.Gui/pull/2664/commits/915ec1bb74cfbf796bc38e8d7251fe527d7f92c9) :)